src/assets/scripts/client/ui/VideoMapController.js

```diff
@@ -0,0 +1,220 @@
+
+        return this;
+    }
+
+    /**
+     * Disable event handlers
+     *
+     * @for TrafficRateController
+     * @method disable
+     * @chainable
+     */
+    disable() {
+        this._eventBus.off(EVENT.AIRPORT_CHANGE, this._onAirportChangeHandler);
+
+        return this;
+    }
+
+    /**
+     * Returns whether the airport selection dialog is open
+     *
+     * @for TrafficRateController
+     * @method isDialogOpen
+     * @return {boolean}
+     */
+    isDialogOpen() {
+        return this.$dialog.hasClass(SELECTORS.CLASSNAMES.OPEN);
+    }
+
+    /**
+    * @for TrafficRateController
+    * @method toggleDialog
+    */
+    toggleDialog() {
+        this.$dialog.toggleClass(SELECTORS.CLASSNAMES.OPEN);
+    }
+
+    /**
+     * Rebuilds the dialog body when the airport is changed.
+     *
+     * @for TrafficRateController
+     * @method onAirportChange
+     */
+    onAirportChange() {
+        this._buildDialogBody();
+    }
+
+    /**
+     * Builds the dialog body
+     *
+     * @for TrafficRateController
+     * @method _buildDialogBody
+     */
+    _buildDialogBody() {
+        this.$dialogBody.empty();
+        const airportModel = AirportController.airport_get();
+        const mapNames = airportModel.mapCollection.getMapNames();
+        this._selectedMaps = airportModel.mapCollection.getVisibleMapNames();
+
+        for (const mapName of mapNames) {
+            const isChecked = this._selectedMaps.includes(mapName);
+            const $formElement = this._buildRow(mapName, mapName, isChecked, this._onChangeSelectedMaps);
+            this.$dialogBody.append($formElement);
+        }
+    }
+
+    /**
+     * Build form element
+     *
+     * @for VideoMapController
+     * @method _buildRow
+     * @param key {string}
+     * @param label {string}
+     * @param checked {boolean} initial value
+     * @param onChangeMethod {function}
+     * @return {jquery|HTML Element}
+     */
+    _buildRow(key, label, checked, onChangeMethod) {
+        const _inputTemplate = checked ? `<input class="form-checkbox" type="checkbox" name="${key}" checked/>` :
+            `<input class="form-checkbox" type="checkbox" name="${key}"/>`;
+        const template = `
+            <div class="form-element">
+                ${_inputTemplate}
+                <label class="form-label"> ${label}</label>
+            </div>`;
+        const $element = $(template);
+        const onChangeHandler = onChangeMethod.bind(this);
+
+        $element.on('change', { name: label }, onChangeHandler);
+
+        return $element;
+    }
+
+    /**
+     * Called when the rate for a flight category was changed
+     *
+     * @for VideoMapController
+     * @method _onChangeSelectedMaps
+     * @param event
+     */
+    _onChangeSelectedMaps(event) {
+        const $target = $(event.target);
+        const value = $target.prop('checked');
+        const map = event.data.name;
+
+        if (value) {
```

Comment:
We try to keep our `if` statements _exclusive_ and really try to avoid using `else`.  Meaning, we like to eject early if a method doesn't have what we need.  The bottom of each method/function should be the happiest of possible paths.

I'm not going to nit-pick on that here, though, but I would encourage possibly splitting this up into two smaller methods (thus allowing for a simple if with `return`) for adding or removing a map.  Not required, but it would make things a little cleaner.